PurchasesTests/Networking/HTTPClientTests.swift

```diff
@@ -33,6 +33,10 @@ class HTTPClientTests: XCTestCase {
         HTTPStubs.removeAllStubs()
     }
 
+}
+
+extension HTTPClientTests {
```

Comment:
Ditto these tests can go to the original class again.

Reply:
Done!